tests/acceptance/test_transaction_comparison.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import absolute_import
+
+import copy
+import pytz
+from sentry.utils.compat.mock import patch
+from datetime import timedelta, datetime
+
+from sentry.testutils import AcceptanceTestCase, SnubaTestCase
+from sentry.testutils.helpers.datetime import before_now, timestamp_format
+from tests.acceptance.test_organization_events_v2 import generate_transaction
+
+FEATURE_NAMES = ["organizations:performance-view"]
+
+
+class TransactionComparison(AcceptanceTestCase, SnubaTestCase):
+    def setUp(self):
+        super(TransactionComparison, self).setUp()
+        self.user = self.create_user("foo@example.com", is_superuser=True)
+        self.org = self.create_organization(name="Rowdy Tiger")
+        self.team = self.create_team(organization=self.org, name="Mariachi Band")
+        self.project = self.create_project(organization=self.org, teams=[self.team], name="Bengal")
+        self.create_member(user=self.user, organization=self.org, role="owner", teams=[self.team])
+
+        self.login_as(self.user)
+
+    def wait_until_loaded(self):
+        self.browser.wait_until_not(".loading-indicator")
+        self.browser.wait_until_not('[data-test-id="loading-placeholder"]')
+
+    @patch("django.utils.timezone.now")
+    def test_transaction_comparison(self, mock_now):
+        mock_now.return_value = before_now().replace(tzinfo=pytz.utc)
+
+        baseline_event_data = generate_transaction()
+        baseline_event = self.store_event(
+            data=baseline_event_data, project_id=self.project.id, assert_no_errors=True
+        )
+        baseline_event_slug = u"{}:{}".format(self.project.slug, baseline_event.event_id)
+
+        regression_event_data = generate_transaction()
+        regression_event_data.update({"event_id": "b" * 32})
+
+        def regress_spans(original_spans):
```

Comment:
Why nest this function? As far as I can tell we don't follow this pattern anywhere else.